src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -902,8 +869,15 @@
   - name: Update platform for the Build step of your Task to Windows (prev Linux).
     text: >
         az acr task update -n MyTask -r MyRegistry --platform Windows
+  - name: Update task's triggers and context for an Azure Container Registry.
+    text: |
+        az acr task update -n hello-world -r MyRegistry -f Dockerfile \\
+            --commit-trigger-enabled true --pull-request-trigger-enabled false \\
```

Comment:
@rosanch as the pr trigger is disabled by default, I would explicitly say in the description that we are disabling a pull-request-trigger because not all users will have this enabled on the tasks, like the commit trigger.